apps/backend/src/modules/auth/auth.controller.ts

```diff
@@ -61,6 +63,30 @@ class AuthController extends BaseController {
 				body: userSignUpValidationSchema,
 			},
 		});
+
+		this.addRoute({
+			handler: this.signIn.bind(this),
+			method: "POST",
+			path: AuthApiPath.SIGN_IN,
+			validation: {
+				body: userSignInValidationSchema,
+			},
+		});
+	}
+
+	public async signIn(
+		options: APIHandlerOptions<{
+			body: UserSignInRequestDto;
+		}>,
+	): Promise<APIHandlerResponse<UserSignUpResponseDto>> {
```

Comment:
Let's create a separate type here even if it fully repeats sign up
```suggestion
	): Promise<APIHandlerResponse<UserSignInResponseDto>> {
```